### ISSUE TITLE
Cannot read property 'length' of undefined' while validating

### ISSUE THREAD

**Issue by johncch** [Microsoft/vscode-tslint#182]
Hello! I've been trying out TSLint and I'm getting this error message. I've been poking around but I'm not sure what's the next step to resolve this.

I'm running
```
MacOS 10.12.4
VSCode: 1.10.2
TSLint: 5.0.0 installed locally via npm
VSCode-TSLint: 0.90
```

The stack trace in question:
```
[Trace - 3:23:34 PM] Received request 'window/showMessageRequest - (4)'.
Params: {
    "type": 1,
    "message": "vscode-tslint: 'Cannot read property 'length' of undefined' while validating: /Users/johncch/Projects/Experiments/three-one/app/examples/wave.ts stacktrace: TypeError: Cannot read property 'length' of undefined\n    
at recordCodeAction (/Users/johncch/.vscode/extensions/eg2.tslint-0.9.0/server/server.js:184:76)\n    at filterProblemsForDocument.forEach.problem (/Users/johncch/.vscode/extensions/eg2.tslint-0.9.0/server/server.js:390:13)\n    
at Array.forEach (native)\n    
at doValidate (/Users/johncch/.vscode/extensions/eg2.tslint-0.9.0/server/server.js:387:60)\n    
at validateTextDocument (/Users/johncch/.vscode/extensions/eg2.tslint-0.9.0/server/server.js:285:27)\n    
at Delayer.d.trigger [as task] (/Users/johncch/.vscode/extensions/eg2.tslint-0.9.0/server/server.js:462:9)\n    
at completionPromise.Promise.then (/Users/johncch/.vscode/extensions/eg2.tslint-0.9.0/server/delayer.js:19:35)",
    "actions": []
}
```

**Comment by slightboy** [comment]
eg2.tslint-0.9.0\server\server.js
line 184: change problem.getFix().replacements to problem.getFix().Replacement

**Comment by egamma** [comment]
@johncch 

>TSLint: 5.0.0 installed locally via npm
You are using the just released tslint 5.0 and tslint 5.0 includes breaking API changes
As a temporary work around you can install the latest tslint 4 version (e.g. npm install tslint@4.5.1)

**Comment by egamma** [comment]
@slightboy thanks 🌷

**Comment by egamma** [comment]
Added support for tslint 5, published as vscode-tslint 0.11.0

**Comment by johncch** [comment]
That fixed my problem - thank you so much!
